diktat-rules/src/test/resources/test/paragraph3/long_line/LongLineCommentExpected.kt

```diff
@@ -7,8 +7,8 @@ fun foo() {
     namesList.forEach { name ->
         if (name == "Nick") {
             namesList.map {
-                it.subSequence(0, 1) // This is
-// another comment inside map
+                // This is another comment inside map
```

Comment:
Could you please provide more tests?
E.g.:
1) some comment, which shouldn't be moved as it too short and can be incline
```kotlin
val a = 5 // it's okay
```
2) some long comment, which should be moved before code, but still remains long and should be splited if possible
Before:
```kotlin
val value = "some string" // long long long comment
```
After, expected
```kotlin
// long long 
// long comment
val value = "some string"
```
Will we handle this situation?

Reply:
No. I thought logic should be as in rule. If we found long line and after fix this line still long we don't fix it again.

Reply:
We have same logic with `BinaryExpression`